_data/KaranamLokesh/KaranamLokesh.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Lokesh Karanam # No longer than 18 characters
+institution: ABV-Indian Institute of Information Technology and Management, Gwalior 🚩 # no longer than 58 characters
```

Comment:
The name of the institution must **not** be longer than 58 characters, yours is **70** characters without the 🚩(flag).